src/SFA.DAS.AANHub.Domain/Entities/StagedApprentice.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace SFA.DAS.AANHub.Domain.Entities
+{
+    public class StagedApprentice
+    {
+        public int Id { get; set; }
+        public string FirstName { get; set; } = null!;
+        public string LastName { get; set; } = null!;
+        public string Email { get; set; } = null!;
+        public DateTime? DateOfBirth { get; set; }
+        public long Uln { get; set; }
+        public long ApprenticeshipId { get; set; }
+        public string?EmployerName { get; set; } = string.Empty;
```

Comment:
Add space after ?

Reply:
Uln and Apprenticeshipid are optional, and will only be set if known.

Reply:
Amended this issue and a few others. Re Alan's comment  re Uln and Apprenticeshipid - Do I make the `long` nullable? Haven't seen a `nullable` `int` or `long` ever!

Reply:
@ravihejmadi yes set those fields to nullable please